PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -146,11 +144,13 @@ class HomeViewModel: NSObject {
 
     /// Fetch the latest data from core data and get the NSFetechedResults Controllers subscribing to updates
     func fetch() {
-        do {
-            try self.recentSavesController.performFetch()
-            try self.recomendationsController.performFetch()
-        } catch {
-            Log.capture(error: error)
+        DispatchQueue.main.async {
+            do {
+                try self.recentSavesController.performFetch()
+                try self.recomendationsController.performFetch()
+            } catch {
+                Log.capture(error: error)
+            }
```

Comment:
Does this affect the thread on which the fetched results controller works? This seems like the opposite of what our changes to what queues we use is.

Reply:
its the oddest stuff. im double checking